website/docs/terms/etl.md

```diff
@@ -0,0 +1,122 @@
+
+## How ETL works
+
+In an ETL process, data is first extracted from a source, transformed, and then loaded into a target data platform. We’ll go into greater depth for all three steps below.
+
+![A diagram depicting the ETL workflow. The diagram starts by depicting raw data being extracted from various example data sources like an email CRM, Facebook Ads platform, a backend database, and Netsuite. Once the data is extracted, the raw data is transformed within the data pipeline via renaming, casting, joining, and enriching. After the data is transformed within the data pipeline, the modeled data is loaded into a data warehouse.](/img/docs/terms/etl/etl-diagram.png)
+
+### Extract
+
+In this first step, data is extracted from different data sources. Data that is extracted at this stage is likely going to be eventually used by end business users to make decisions. Some examples of these data sources include:
+
+- Ad platforms (Facebook Ads, Google Ads, etc.) 
+- Backend application databases 
+- Sales CRMs
+- And more!
+
+To actually get this data, data engineers may write custom scripts that make Application Programming Interface (API) calls to extract all the relevant data. Because making and automating these API calls gets harder as data sources and data volume grows, this method of extraction often requires strong technical skills. In addition, these extraction scripts also involve considerable maintenance since APIs change relatively often. Data engineers are often incredibly competent at using different programming languages such as Python and Java. Data teams can also extract from these data sources with open source and Software as a Service (SaaS) products.
+
+### Transform
+
+At this stage, the raw data that has been extracted is normalized and modeled. In ETL workflows, much of the actual meaningful business logic, metric calculations, and entity joins tend to happen further down in a downstream BI platform. As a result, the transformation stage here is focused on data cleanup and normalization – renaming of columns, correct casting of fields, timestamp conversions. 
+
+To actually transform the data, there’s two primary methods teams will use:
+
+- **Custom solutions**: In this solution, data teams (typically data engineers on the team), will write custom scripts and create automated pipelines to transform the data. Unlike ELT transformations that typically use SQL for modeling, ETL transformations are often written in other programming languages such as Python or Scala. Data engineers may leverage technologies such as Apache Spark or Hadoop at this point to help process large volumes of data. 
+- **ETL products**: There are ETL products that will extract, transform, and load your data in one platform. These tools [(link to table below)](#etl-tools) often involve little to no code and instead use Graphical User Interfaces (GUI) to create pipelines and transformations.
```

Comment:
Should have made this more obvious in my google doc for this, but if we could link on These tools instead of explicitly saying "link to table below" that would be fantastic:

[These tools]((#etl-tools)